javascript-typing-tutor/main.js

```diff
@@ -0,0 +1,14 @@
+var current = document.querySelectorAll("span");
+var i = 0;
+var counter = 0;
+var results = (100 / counter);
```

Comment:
I assume you'll be using this if you intend to add the accuracy, but if not, I would remove all unused variables / code. If you plan on using this, you can ignore this comment

Reply:
yeah, I'll delete them for the submit, but I'll be messing with them later.